### DISCUSSION TITLE
Feature request:

### DISCUSSION THREAD

**Discussion by MBloemsma** [algolia/instantsearch#5914] [Ideas]
Create a widget for filtering on location, in my case an input field to enter a postal code and a select where you can edit to distance from this postal code.

**Comment by Haroenv** [comment]
As Algolia Places no longer exists, this widget: https://github.com/algolia/instantsearch/blob/master/packages/instantsearch.js/src/widgets/places/places.ts also no longer works, but you can most likely take the code that's there, and edit it for another location provider, like placekit, mapbox or google places
